Update README.md

mkurisoo opened the pull request:
Several updates: Commands were changed to represent actual directory names (directory names listed appeared outdated); more steps were added for clarity; wording and formatting was changed in some areas.

mkurisoo committed:
Update README.md

Several updates: Commands were changed to represent actual directory names (directory names listed appeared outdated); more steps were added for clarity; wording and formatting was changed in some areas.

mkurisoo commented:
I have read the CLA Document and I hereby sign the CLA

mkurisoo committed:
Further updated README.md

Simplified and re-ordered some of the steps detailed in the update committed on 8/5/22.

psfoley commented on openfl-tutorials/interactive_api/Tensorflow_MNIST/README.md:
````diff
@@ -2,43 +2,67 @@
+   ```sh
+   source venv/bin/activate
+   ```
+   - Ensure proxies are set. If you do not have sudo permissions on your machine, ensure your proxies are set in all terminals.
````
Proxies are rarely necessary for users. Would recommend changing this wording to "If you are in a network environment with a proxy, ensure proxy environment variables are set in each of your terminals.". Would also recommend removing the 2nd sentence because sudo permissions are not required for this, and could be confusing.

mkurisoo replied:
Thank you for the suggestion! I agree that the wording you suggested sounds less confusing. The only reason I mentioned sudo permissions was because when I was learning how to set up the proxies, a teammate mentioned that if I had sudo permissions I could set it up so that I wouldn't have to consistently type the proxy info into every terminal. I can see how adding that may confuse users though. I will revise it.

mkurisoo committed:
WIP: Updates to README

Fixed line describing proxy settings, as suggested by reviewer psfoley. I have labeled this as "WIP" since I have ideas for additional changes which I will discuss with the team.
